src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -1,9 +1,8 @@
 import { LightningElement, track, api } from 'lwc';
 import GeLabelService from 'c/geLabelService';
+import { registerListener, unregisterListener, fireEvent } from 'c/pubsubNoPageRef';
```

Comment:
It looks like maybe registerListener and unregisterListener are unused in this component?  If that is correct can they be removed?

Reply:
Removed!